### PR TITLE
Update INSTALL.md for MacOS

### PR DESCRIPTION
**Pull request by sisuresh**
# Description

Related to https://github.com/stellar/stellar-core/issues/3707

# Checklist
- [ ] Reviewed the [contributing](https://github.com/stellar/stellar-core/blob/master/CONTRIBUTING.md#submitting-changes) document
- [ ] Rebased on top of master (no merge commits)
- [ ] Ran `clang-format` v8.0.0 (via `make format` or the Visual Studio extension)
- [ ] Compiles
- [ ] Ran all tests
- [ ] If change impacts performance, include supporting evidence per the [performance document](https://github.com/stellar/stellar-core/blob/master/performance-eval/performance-eval.md)

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ In order to install the llvm (clang) toolchain, you may have to follow instructi
 When building on OSX, here's some dependencies you'll need:
 - Install xcode
 - Install [homebrew](https://brew.sh)
-- `brew install libsodium libtool autoconf automake pkg-config libpq openssl parallel ccache bison sed perl`
+- `brew install libsodium libtool autoconf automake pkg-config libpq openssl parallel ccache bison sed perl coreutils`
```
Comment (tsachiherman) [inline INSTALL.md:114]: there is no `sed` package. using `gnu-sed` instead.

Comment (sisuresh) [reply INSTALL.md]: updated

**Review by tsachiherman — APPROVED** [review]

**Comment by sisuresh** [comment]
@tsachiherman did you have to update your path for `gnu-sed`?

**Comment by tsachiherman** [comment]
> @tsachiherman did you have to update your path for `gnu-sed`?

No; it was working right away.

**Comment by MonsieurNicolas** [comment]
r+ b226ec56ee26952dd312cbeb99f8fa1e44e33eae

**Pull request merged**